tee-worker/app-libs/stf/src/getter.rs

```diff
@@ -278,6 +280,8 @@ impl ExecuteGetter for PublicGetter {
 				} else {
 					None
 				},
+			PublicGetter::all_id_graph_hash =>
+				Some(IdentityManagement::all_id_graph_hash().encode()),
```

Comment:
May this cause problems if somebody starts to flood node with such getters ?

Reply:
Then it's not (only) a problem of this one: it applies to any RPC interface: getter, trusted getter or trusted calls. It also applies to any relay/parachains - you could spam costly requests like `state_getKeys`